plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/BaseJdbcConnectorTest.java

```diff
@@ -325,7 +326,7 @@ public void testCaseSensitiveAggregationPushdown()
                 "(a_string varchar(1), a_char char(1), a_bigint bigint)",
                 ImmutableList.of(
                         "'A', 'A', 1",
-                        "'B', 'B', 2",
+                        "'B', 'B', 1",
```

Comment:
Is this required? The number was originally only added because it helped to remind the sort order of strings according to Trino. The bigint column doesn't have any actual purpose for the test.

Reply:
I think yes, now we are use it to test the `sum(DISTINCT)`